archive_communication/api-requirements.md

```diff
@@ -0,0 +1,324 @@
+# API Requirements
+
+This document outlines some general requirements for APIs that would be necessary
```

Comment:
Should note that this currently only has general requirements and BCDC asks of NeMO.